.github/workflows/nightlydeploy.yml

```
@@ -24,6 +24,11 @@ jobs:
       #     fetchReviewers: true
 
       - uses: actions/checkout@v3
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: ${{ matrix.java }}
```

Copy-and-paste error when copying from tests.yml?
```suggestion
          java-version: 17
```